client/src/components/Tags.tsx

```diff
@@ -17,7 +17,10 @@ export default function GenerateTags(tags: Array<string>) {
 /* Structure for each tag item -- colored "bubble" */
 const TagItem = styled(Paper)(({ theme }) => ({
   ...theme.typography.caption,
-  padding: theme.spacing(0.5, 2, 0.5, 2),
+  paddingLeft: 8,
+  paddingRight: 8,
+  paddingTop: 4,
+  paddingBottom: 4,
```

Comment:
```suggestion
  padding: 4 8 4 8
```
https://developer.mozilla.org/en-US/docs/Web/CSS/padding